lib/sisimai/data.rb

```diff
@@ -361,13 +359,13 @@ def self.make(data: nil, **argvs)
             textasargv = textasargv.gsub(/\A[ ]/, '')
             softorhard = Sisimai::SMTP::Error.soft_or_hard(o.reason, textasargv)
 
-            if softorhard.size > 0
-              # Returned value is "soft" or "hard"
-              o.softbounce = (softorhard == 'soft') ? 1 : 0
-            else
-              # Returned value is an empty string
-              o.softbounce = (-1)
-            end
+            o.softbounce = if softorhard.size > 0
+                             # Returned value is "soft" or "hard"
+                             (softorhard == 'soft') ? 1 : 0
```

Comment:
__[rubocop]__
* Style/TernaryParentheses: Omit parentheses for ternary conditions.

<!-- Comment by [SideCI](https://sideci.com) -->